src/rez_pip/cli.py

```diff
@@ -163,6 +163,11 @@ def _run(args: argparse.Namespace, pipArgs: typing.List[str], pipWorkArea: str)
         args.python_version, packageFamily="python"
     )
 
+    if not pythonVersions:
+        raise RuntimeError(
+            f"No python package found on system for python-{args.python_version}"
```

Comment:
```suggestion
```suggestion
        raise rez_pip.exceptions.RezPipError(
            f'No "python" package found within the range {args.python_version!r}.'
```
By using `RezPipError`, the error will be nicely integrated and will be consistent with the other errors raised by rez-pip. I also tweaked the error message a bit.

Reply:
done !